src/components/Task/Difficulty/DifficultyLabel/DifficultyLabel.scss

```diff
@@ -1,14 +1,23 @@
 @import './src/css/styles';
 
-.difficulty-label {
+.difficulty-label-container {
+  position: relative;
+  width: 5rem;
+}
+
+.difficulty-label-item {
   align-items: center;
   border-radius: 1rem;
   border-style: solid;
   border-width: 1px;
   color: $color-dark;
-  display: flex;
+  display: inline-flex;
```

Comment:
why not flex?

Reply:
> Size shouldn't be fixed. The problem with fixed sizes is the length of the words and the difference in other languages.

but imo the design is broken if everything is aligned differently

Reply:
doesnt work for me correctly. So everything is aligned correctly